src/main/java/bio/terra/cli/command/app/passthrough/Nextflow.java

```diff
@@ -16,11 +19,17 @@
 
   @CommandLine.Mixin WorkspaceOverride workspaceOption;
   @CommandLine.Unmatched private List<String> command = new ArrayList<>();
+  private static final Logger logger = LoggerFactory.getLogger(Nextflow.class);
 
-  /** Pass the command through to the CLI Docker image. */
+  /**
+   * Enable the user's pet SA in the workspace context, then pass the command through to the CLI
+   * Docker image.
+   */
   @Override
   protected void execute() {
     workspaceOption.overrideIfSpecified();
+    String petEmail = Workspace.enablePet();
+    logger.info("Enabled usage of for pet {} for Nextflow", petEmail);
```

Comment:
nit: I think the logging should happen in either the `Workspace` or `WorkspaceManagerService` class instead of here in the command.